spec/services/mentions/create_all_spec.rb

```diff
@@ -1,88 +1,140 @@
 require "rails_helper"
 
-RSpec.describe Mentions::CreateAll, type: :service do
-  let(:user)        { create(:user) }
-  let(:user2)       { create(:user) }
-  let(:article)     { create(:article, user_id: user.id) }
-  let(:comment)     { create(:comment, user_id: user2.id, commentable: article) }
-  let(:comment2)    do
-    create(
-      :comment,
-      body_markdown: "Hello @#{user.username}, you are cool.",
-      user_id: user2.id,
-      commentable: article,
-    )
-  end
-
-  it "creates mention if there is a user mentioned and if the user doenst own the comment" do
-    comment.body_markdown = "Hello @#{user.username}, you are cool."
-    comment.save
-    described_class.call(comment)
-    expect(Mention.all.size).to eq(1)
+RSpec.shared_examples "valid notifiable and no mentions" do
```

Comment:
I refactored a bunch of tests into shared examples since the behavior for creating mentions on comments is the same as for creating mentions on articles; I opted to keep these shared examples within the same file because I personally found it much easier to read, but I'm happy to pull them out into separate files if folks would prefer that.